src/objecttypes/scss/admin/_admin_theme.scss

```diff
@@ -36,6 +36,12 @@ DO NOT PUT ANY TARGET APP-SPECIFIC RULES HERE.
 
   --object-tools-bg: #888;
   --object-tools-hover-bg: #{$color_dark};
+
+  // version styling
+  --version-fg: #{$color-secondary-dark};
```

Comment:
Maybe we should make this a bit darker? It's not very readable imo

![image](https://github.com/maykinmedia/objecttypes-api/assets/29249171/78afc606-fb79-4179-99ff-7d74d8b2d9cd)

Reply:
Now that it reuses another variable (same as the welcome message):
![image](https://github.com/maykinmedia/objecttypes-api/assets/5655384/a34ee958-3da3-414f-99fa-8007365bde59)